src/tests/ftest/cart/rpc/cart_rpc_one_node_with_swim_notification_on_rank_eviction.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2018-2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+
+from __future__ import print_function
+
+import sys
+import time
+
+from avocado       import Test
+from avocado       import main
+
+sys.path.append('./util')
+
+# Can't all this import before setting sys.path
+# pylint: disable=wrong-import-position
+from cart_utils import CartUtils
+
+class CartRpcOneNodeSwimNotificationOnRankEvictionTest(Test):
+    """
+    Runs basic CaRT RPC tests
+
+    :avocado: tags=all,cart,pr,rpc,one_node,swim_rank_eviction
+    """
+    def setUp(self):
+        """ Test setup """
+        print("Running setup\n")
+        self.utils = CartUtils()
+        self.env = self.utils.get_env(self)
+
+    def tearDown(self):
+        """ Test tear down """
+        print("Run TearDown\n")
+
+    def test_cart_rpc(self):
+        """
+        Test CaRT RPC
+
+        :avocado: tags=all,cart,pr,rpc,one_node,swim_rank_eviction
+        """
+        srvcmd = self.utils.build_cmd(self, self.env, "test_servers")
+
+        try:
+            srv_rtn = self.utils.launch_cmd_bg(self, srvcmd)
+        # pylint: disable=broad-except
+        except Exception as e:
+            self.utils.print("Exception in launching server : {}".format(e))
+            self.fail("Test failed.\n")
+
+        # Verify the server is still running.
+        if not self.utils.check_process(srv_rtn):
+            procrtn = self.utils.stop_process(srv_rtn)
+            self.fail("Server did not launch, return code %s" \
+                       % procrtn)
+
+        cli_arg = self.params.get("test_clients_arg", '/run/tests/*/')
+        for index in range(len(cli_arg)):
+            clicmd = self.utils.build_cmd(
+                self, self.env, "test_clients", index=index)
+            self.utils.launch_test(self, clicmd, srv_rtn)
+
+            # Wait for servers to propagate SWIM status
+            time.sleep(10)
```

Comment:
i dont think we should be sleeping after each client command; instead sleep itself should be part of command sequence or we need some identifier in client commands to only do sleep after we killed the server

Reply:
I think the [`--holdtime` option to `test_group_np_cli`](https://github.com/daos-stack/daos/blob/5f3db26153dd1451dffd005b00a29e02704ddda6/src/tests/ftest/cart/test_group_np_common.h#L299) can do the sleep after sending the shutdown RPC.  

b13f138

Reply:
We shouldn't sleep after *every* client command, but we need to sleep after sending the shutdown RPC (and before checking swim status).  I'm guessing this could be due to a race condition between the client swim verification check and the swim status propagating to all the servers.  Here's a patch to sleep after just one specific client commands. 5d6db7f05752b683e87d0d12522e97e1244a60dc